rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -172,6 +172,16 @@ void main_window::Init()
 
 	// Fix possible hidden game list columns. The game list has to be visible already. Use this after show()
 	m_gameListFrame->FixNarrowColumns();
+
+#if defined(_WIN32) || defined(__linux__)
+	if (guiSettings->GetValue(gui::m_check_upd_start).toBool())
+	{
+		updater.check_for_updates(true, this);
+	}
+#else
+	// Disables menu for other oses
+	ui->updateAct->setEnabled(false);
```

Comment:
Instead of not showing this you should probably spawn a messagebox that tells you that it is not supported in that OS.